massa-network-exports/src/commands.rs

```diff
@@ -138,20 +135,41 @@ pub enum NodeEventType {
 #[derive(Clone, Debug)]
 pub struct NodeEvent(pub NodeId, pub NodeEventType);
 
+/// Ask for the info about a block.
+#[derive(Debug, Clone, Default, Serialize, Deserialize)]
+pub enum AskForBlocksInfo {
+    /// The info about the block is required(list of operations ids).
+    #[default]
+    Info,
+    /// The actual operations are required.
+    Operations(OperationIds),
+}
+
+/// Reply with the info about a block.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub enum ReplyForBlocksInfo {
+    /// The info about the block is required(list of operations ids).
+    Info(OperationIds),
+    /// The actual operations required.
+    Operations(OperationIds),
```

Comment:
I don't understand where the actual operations are sent.
Shouldn't this be `Operations(Vec<WrappedOperation>),` ?

Reply:
@gterzian  see this comment and the ones above

Reply:
@damip The actual operations are obtained from storage at https://github.com/massalabs/massa/pull/2862/files#diff-2350ca2a3fef681c7029e38fa91036024b30b38322f5807357e98ec037030873R200

Reply:
Actually I guess we can change this to use the new storage system?Will file an issue.